webpack.common.js

```diff
@@ -22,5 +24,28 @@ module.exports = {
         use: ["style-loader", "css-loader"]
       }
     ]
-  }
+  },
+  plugins: [
+    new HandlebarsTemplatePlugin({
+      templatesPath: path.resolve(__dirname, "templates"),
+      helpers: {
+        /**
+         * Register a handlebars helper that prepends the base asset path.
+         * Useful for things like placing assets on a CDN and cache busting.
+         * Example:
+         * input: <img src="{{asset "asset.png"}}"/>
+         * output: <img src="https://cdn.mysite.com/asset.png?c="/>
+         */
+        asset: assetPath => {
```

Comment:
Just `asset` feels like a weird name for this. Maybe `base_asset_path` to patch the envar?